README.md

```diff
@@ -105,7 +105,7 @@ Sent from Mission Control to instruct the rover to drive like a tank with a spec
 - `left` - left component in [-1.0, 1.0], where positive means drive forward on the left side and negative means drive backward on the left side
 - `right` - right component in [-1.0, 1.0], where positive means drive forward on the right side and negative means drive backward on the right side
 
-## Holonomic Drive Request
+## Holonomic Drive Request ver 2023.0.0
```

Comment:
The version is defined at the top of the README, not on a per-message basis, can you remove this version and update the one above?